load_balancer.go

```diff
@@ -87,24 +89,32 @@ func (odbi *ovnDBImp) lbAddImp(name string, vipPort string, protocol string, add
 		Row:      row,
 		UUIDName: namedUUID,
 	}
-
-	mutateUUID := []libovsdb.UUID{{namedUUID}}
-	mutateSet, err := libovsdb.NewOvsSet(mutateUUID)
-	if err != nil {
-		return nil, err
-	}
-
-	mutation := libovsdb.NewMutation("load_balancer", opInsert, mutateSet)
-	// TODO: Add filter for LS name
-	condition := libovsdb.NewCondition("name", "!=", "")
-
-	mutateOp := libovsdb.Operation{
-		Op:        opMutate,
-		Table:     tableLogicalSwitch,
-		Mutations: []interface{}{mutation},
-		Where:     []interface{}{condition},
+	operations = append(operations, insertOp)
+
+	// Do not mutate all lswitches; instead only for the ones which are meant
+	// to serve LB traffic.
+	LbLswitches := os.Getenv("LB_LSWITCHES")
```

Comment:
Environment variable is not the right way to pass parameter to API. The API LBAdd should simply add the LB, without attaching to any LS, and we should have a separate API to add/remove a LB to/from LS(es).

Reply:
yes, we need to follow nbctl
```
lr-lb-add ROUTER LB       add a load-balancer to ROUTER
  lr-lb-del ROUTER [LB]     remove load-balancers from ROUTER
  lr-lb-list ROUTER         print load-balancers
  ls-lb-add SWITCH LB       add a load-balancer to SWITCH
  ls-lb-del SWITCH [LB]     remove load-balancers from SWITCH
  ls-lb-list SWITCH         print load-balancers
```

Reply:
> Environment variable is not the right way to pass parameter to API. The API LBAdd should simply add the LB, without attaching to any LS, and we should have a separate API to add/remove a LB to/from LS(es).

Sure. Done! PTAL at revised PR

Reply:
> yes, we need to follow nbctl
> 
> ```
>   ls-lb-add SWITCH LB       add a load-balancer to SWITCH
>   ls-lb-del SWITCH [LB]     remove load-balancers from SWITCH
>   ls-lb-list SWITCH         print load-balancers
> ```

@vtolstov : Sure we can do that. I added the code in LB instead instead of changing logical_switch.go. Let me know if we still want to prefer nbctl way!

Reply:
@vtolstov : Revised as per nbctl format. PTAL 
CC @hzhou8